Code/Framework/AzToolsFramework/AzToolsFramework/Viewport/ViewBookmarkLoader.cpp

```diff
@@ -21,6 +21,7 @@ namespace AzToolsFramework
     static constexpr const char* s_viewBookmarksRegistryPath = "/O3DE/ViewBookmarks/";
     static constexpr const char* s_localBookmarksKey = "LocalBookmarks";
     static constexpr const char* s_lastKnownLocationKey = "LastKnownLocation";
+    static constexpr int s_numOfDefaultLocationsInLevel = 12;
```

Comment:
suggestion: Might be worth adding a comment that this is temporary until additional UI is added to expose the fields